bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/Parameters.scala

```diff
@@ -10,7 +10,9 @@ final case class ObservingConditions(
                       cc: CloudCover,
                       wv: WaterVapor,
                       sb: SkyBackground,
-                      airmass: Double)
+                      airmass: Double,
+                      exactiq: Double,
```

Comment:
I'm wondering if we could leave `ImageQuality` as it is and instead make these `airmass: Option[Double]` where if it is defined we take it as  an indication to use that value instead of the one from iq.
I fear a bit changing `ImageQuality` enum as it is being iterated in quite a few places.
I think this change would allow the OT to remain as it is and still let the web itc to accept exact values

Reply:
That's a good point about the OT.  When do we want these changes to go into production?  Because we use TRPC for OT <-> ITC communication, unless the update is made along with the December release we'll either need to require an OT update or else make sure we provide a constructor that allows this original call to succeed:

```scala
new ObservingConditions(
     iq.selection.item,
     cc.selection.item,
     wv.selection.item,
     sb.selection.item,
     am.selection.item)
```

along with support for maintaining binary compatibility via the correct `serialVersionUID` and a `readObject` implementation.

Reply:
Is that because `ObservingConditions` is passed in the TRPC request?
I think this update could be published to production on the web itc without affecting the OT and then we'd deploy both OT and SPDB to production together in the next release.

Reply:
Oh, okay the web ITC communicates with a different server (and not the SPDB)?

Reply:
As far as I understand it the itc web has a web backend that serves the web pages and does calculations and that's the one we deploy to the itc server. In the meanwhile the OT talks directly to the itc running in spdb over TRPC